docs/job/push-to-registries.md

```diff
@@ -25,6 +25,7 @@ workflows:
       - architect/push-to-registries:
           context: architect
           name: push-to-registries
+          image: giantswarm/IMAGE_NAME
```

Comment:
Since `image` it's optional now, can we please remove it here, so that this works as a minimal example?